base/lthooks.dtx

```diff
@@ -6786,8 +6786,6 @@
 %
 % \changes{v1.0s}{2021/09/28}
 %                {Correct usage of older \cs{@@_if_file_hook:wTF} (gh/675)}
-% \changes{v1.1h}{2024/01/24}
-%                {Correct usage of older \cs{@@_if_file_hook:wTF} (gh/1243)}
```

Comment:
This deletion looks wired.

Reply:
I think it's spurious: will vanish when properly rebased